hw/xbox/dsp/dsp_dma.c

```diff
@@ -137,9 +179,51 @@ static void dsp_dma_run(DSPDMAState *s)
             assert(false);
         }
 
+#ifdef DEBUG
+        char dsp_space_name = '?';
+        if (mem_space == DSP_SPACE_X) {
+            dsp_space_name = 'x';
+        } else if (mem_space == DSP_SPACE_Y) {
+            dsp_space_name = 'y';
+        } else if (mem_space == DSP_SPACE_P) {
+            dsp_space_name = 'p';
+        }
+#endif
+
+        DPRINTF("dsp dma block 0x%x (%s)\n"
+                "    next-block 0x%x%s\n"
+                "    control 0x%06x:\n"
+                "        dsp-interleave %d\n"
+                "        buffer-offset-writeback %d\n"
+                "        buffer 0x%x (%s)\n"
+                "        unk9 %d\n"
+                "        sample-format 0x%x (%s)\n"
+                "        dsp-step 0x%x\n"
+                "    sample-count 0x%x\n"
+                "    block-count 0x%x channel-count %d\n"
```

Comment:
nit: "sample-count" is only valid if `dsp_interleave` is false; "block-count" and "channel-count" are only valid if `dsp_interleave` is true